packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -58,6 +58,117 @@ const {
 const { fakeFileFactory, fakeGranuleFactoryV2 } = require('../../../lib/testUtils');
 const Granule = require('../../../models/granules');
 
+// FUTURE:
+// 1. 'created_at' is updated during PUT/PATCH
+// 2. 'published' defaults to false if not provided in the payload
+// 3. 'duration' comes from the workflow and will be reset on update
+// 4. 'product_volume' comes from a files object on the payload, which may not exist
+//   in the case of partial granule updates
+const cumulusMessageOmitList = [
+  'cumulus_id',
+  'updated_at',
+  'created_at',
+  'published',
+  'timestamp',
+  'duration',
+  'product_volume',
+];
+
+// FUTURE:
+// 1. 'created_at' is updated during PUT/PATCH
+// 2. 'published' defaults to false if not provided in the payload
+const apiOmitList = [
+  'cumulus_id',
+  'updated_at',
+  'created_at',
+  'published',
```

Comment:
Is there a good reason we don't explicitly validate this doesn't change in our tests?

Reply:
`published` does change though. If you have an existing granule where `published: true` and update that granule _without_ a `published` key in the payload, the resulting granule will be `published: false`. 

This behavior is captured in the Google Doc and will need to be updated. Only reason I'm not addressing here is that it doesn't result in a sync issue with our datastores.

Did I answer your question? I feel like you might have meant something else.

Reply:
No, that answers the question.